frontend/test/simulate/cluster_mutation_test.go

```diff
@@ -98,14 +98,22 @@ func (tt *clusterMutationTest) runTest(t *testing.T) {
 	clusterClient := tt.testInfo.Get20240610ClientFactory(tt.subscriptionID).NewHcpOpenShiftClustersClient()
 	_, mutationErr := clusterClient.BeginCreateOrUpdate(ctx, tt.resourceGroupName, *toCreate.Name, *toCreate, nil)
 
-	if tt.genericMutationTestInfo.isUpdateTest() {
+	if tt.genericMutationTestInfo.isUpdateTest() || tt.genericMutationTestInfo.isPatchTest() {
 		require.NoError(t, mutationErr)
 		require.NoError(t, MarkOperationsCompleteForName(ctx, tt.testInfo.DBClient, tt.subscriptionID, ptr.Deref(toCreate.Name, "")))
+	}
 
+	switch {
```

Comment:
Doesn't smell like `func (tt *clusterMutationTest) runTest(t *testing.T) {` is factored well, but if this is expedient for you, sure. Probably some other factoring that separates these out would be easier